### DISCUSSION TITLE
My account shows I was "approved" on the 25/3/24 – however, I have received rejection emails

### DISCUSSION THREAD

**Discussion by memr330** [community/community#116491] [GitHub Education]
### Hi everyone,

My account shows I was "approved" on the 25/3/24 – however, I have received rejection emails. I kept getting rejected as I'm a distance student and my documents were not being picked up as indicating I was an online student. On my log in it says I have been approved but am waiting on access, however, all my emails are rejections. 

- How can I clarify my status? 
- Am I able to send multiple documents via email to confirm my status as the system does not seem to like the format my uni uses?

TIA!

**Comment by samus-aran** [comment]
Hi @memr330 👋 

Yes I suspect it was sent in error. The latest application status should be at the top of your application page. It's currently Approved so you still have 72 hours since this Approved state before you receive your academic benefits. 👍
